wordpress-mobile/WordPress-Android
# Issue/11760 reader empty view ui

Fixes #11760 
This PR changes the Reader UI for empty views on the Following tab and the Followed Site tabs. The copy has been updated to reflect the requested design changes. 

![Screenshot_20200504-095759_WordPress Beta](https://user-images.githubusercontent.com/506707/80974136-55d55a80-8dee-11ea-88f1-5ea837d94d89.jpg)
![Screenshot_20200504-095855_WordPress Beta](https://user-images.githubusercontent.com/506707/80974148-58d04b00-8dee-11ea-83fd-2d1100152e65.jpg)
![Screenshot_20200504-095916_WordPress Beta](https://user-images.githubusercontent.com/506707/80974153-5b32a500-8dee-11ea-8283-4c941c5d137d.jpg)
![Screenshot_20200504-095922_WordPress Beta](https://user-images.githubusercontent.com/506707/80974160-5cfc6880-8dee-11ea-8c16-f945d1339c93.jpg)

To Test
(A) Light mode
- Log into an account that is not following any sites
- Set app to light mode
- Navigate to the reader tab
- Tab on the Following tab 
- The updated empty view should be visible
- Tap on the  Manage Tags & Sites Settings Cog
- Tap on the Followed Sites tab
- The updated empty view should be visible

(B) Set app to dark mode and repeat above steps


PR submission checklist:
